Xamarin.Forms.Core/Items/ScrolledEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core.Items
+{
+	public class ScrolledEventArgs : EventArgs
+	{
+		public double HorizontalDelta { get; private set; }
+
+		public double VerticalDelta { get; private set; }
+
+		public double HorizontalOffset { get; private set; }
+
+		public double VerticalOffset { get; private set; }
+
+		public int FirstVisibleItemIndex { get; private set; }
+
+		public int CenterItemIndex { get; set; }
+
+		public int LastVisibleItemIndex { get; private set; }
+
+		public ScrolledEventArgs(double horizontalDelta, double verticalDelta, double horizontalOffset, double verticalOffset, int firstVisibleItemIndex, int centerItemIndex, int lastVisibleItemIndex)
```

Comment:
that's a hell of a signature ! shouldn't the calling code set the properties instead ?

Reply:
I'm fine with this. I was trying to match the existing constructor in the other class since it didn't make public setters available.